### PR TITLE
added logger; change system.out to logger in main

### PR DESCRIPTION
**Pull request by TimoxaV**

### CODE REVIEW

**Review by AnaLanda — COMMENTED** [review]
Looks good)

File: src/main/java/com/cinema/Main.java [hunk 1]
```diff
@@ -16,32 +16,34 @@
 import com.cinema.service.ShoppingCartService;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import org.apache.log4j.Logger;
 
 public class Main {
+    private static Logger logger = Logger.getLogger(Main.class);
     private static Injector injector = Injector.getInstance("com.cinema");
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]:
```suggestion
    private static final Logger logger = Logger.getLogger(Main.class);
    private static final Injector injector = Injector.getInstance("com.cinema");
```

File: src/main/java/com/cinema/Main.java [hunk 2]
```diff
@@ -68,25 +70,25 @@ public static void main(String[] args) throws AuthenticationException {
         sessionFastAndFurious2.setShowTime(LocalDateTime.now().minusDays(2));
         sessionService.add(sessionFastAndFurious2);
 
-        System.out.println("------------------Find Fast And Furious Sessions--------------");
+        logger.info("------------------Find Fast And Furious Sessions--------------");
         sessionService.findAvailableSessions(fastAndFurious.getId(),
                 LocalDate.now())
-                .forEach(System.out::println);
+                .forEach(logger::info);
         System.out.println("------------------Find Sully Sessions--------------------------");
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]: replace this

File: src/main/java/com/cinema/Main.java [hunk 3]
```diff
@@ -68,25 +70,25 @@ public static void main(String[] args) throws AuthenticationException {
...
+                .forEach(logger::info);
 
-        System.out.println("------------------Users Check--------------------------");
+        logger.info("------------------Users Check--------------------------");
         AuthenticationService authService =
                 (AuthenticationService) injector.getInstance(AuthenticationService.class);
         authService.register("new1@gmail.com", "1234");
         authService.register("new2@gmail.com", "1234");
         User new1 = authService.login("new1@gmail.com", "1234");
         User new2 = authService.login("new2@gmail.com", "1234");
```
Comment (KseniiaMakarova) [inline src/main/java/com/cinema/Main.java]: let's wrap call to `login()` method in try-catch and add log.warn with the informative message in `catch` block

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]

File: src/main/java/com/cinema/Main.java (hunk 1 above)
Comment (TimoxaV) [reply src/main/java/com/cinema/Main.java]: done

File: src/main/java/com/cinema/Main.java (hunk 2 above)
Comment (TimoxaV) [reply src/main/java/com/cinema/Main.java]: done

File: src/main/java/com/cinema/Main.java (hunk 3 above)
Comment (TimoxaV) [reply src/main/java/com/cinema/Main.java]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**